R/Rules-class.R

```diff
@@ -1551,7 +1549,8 @@ IncrementsOrdinal <- function(grade, rule) {
 #'
 #' [`Stopping`] is a class for stopping rules.
 #'
-#' @slot report_label (`string`)\cr a label for the stopping report. The meaning
+#' @slot
```

Comment:
seems the slot name is missing here

Reply:
Hmmm.  Not one of my changes, but fixed.